.github/workflows/eslint.yml

```diff
@@ -0,0 +1,15 @@
+name: eslint
+
+on:
+  pull_request:
+  push:
```

Comment:
```suggestion
  push:
  workflow_dispatch:
```

Reply:
Having a button to run tests as maintainers of the repo on the main branch is useful to verify that tests succeed on main branch if they fail for a weird reason on a PR.

I always add this to my workflows unless I see a reason it could be problematic.